vault/templates/deployment.yaml

```diff
@@ -176,12 +176,16 @@ spec:
         - name: vault-root
           emptyDir: {}
         - name: vault-file
-        {{- if .Values.persistence.enabled }}
+        {{- if .Values.persistence.enabled  }}
           persistentVolumeClaim:
             claimName: {{ template "vault.fullname" . }}-pvc
         {{- else }}
+          {{- if .Values.persistence.hostPath  }}
           hostPath:
             path: /opt/vault
```

Comment:
```suggestion
            path: {{ .Values.persistence.hostPath }}
```

Reply:
I'm not sure about this, but the value is given as a string and not a boolean, so I assumed it should be the value here.